Octokit.Reactive/IObservableGitHubClient.cs

```diff
@@ -16,6 +16,7 @@ public interface IObservableGitHubClient : IApiInfoProvider
         IObservableReleasesClient Release { get; }
         IObservableSshKeysClient SshKey { get; }
         IObservableUsersClient User { get; }
+        [System.Obsolete("The Notifications API belongs under the Activities client")]
```

Comment:
:lipstick: Perhaps make this message a bit more concise:

> Notifications are now available under the Activities client. This will be removed in a future update.